movie-app/src/components/MovieBanner.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-undef */
+import React, { useState } from 'react';
+import Carousel from 'react-material-ui-carousel';
+import useFetch from '../hooks/useFetch';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Rating from '@mui/material/Rating';
+
+const MovieBanner = () => {
+  const [url] = useState(`/api/fetch-now-playing`);
+  const {
+    data: { results: movie },
+    error,
+    isLoading,
+  } = useFetch(url);
+
+  const item = movie && movie.slice(0, 5);
+
+  console.log(movie);
+  return (
+    <Container>
+      {isLoading ? (
+        'Loading...'
+      ) : (
+        <Box>
+          {movie ? (
+            <Carousel navButtonsAlwaysVisible={true} duration={1000}>
+              {item.map((movie) => (
+                <Box key={movie.id} sx={{ width: '100%', maxHeight: '500px' }}>
+                  <Typography
+                    sx={{
+                      position: 'absolute',
+                      top: '5%',
+                      left: '10%',
+                      bgcolor: '#E0E0E0',
+                      color: '#000',
+                      borderRadius: '20px',
+                    }}
+                    p={1}
+                  >
+                    Movie of the Day
+                  </Typography>
+
+                  <Typography
+                    sx={{ position: 'absolute', top: '20%', left: '10%' }}
+                    variant="h3"
+                    component="h1"
+                  >
+                    {movie.title ? movie.title : movie.name}
+                  </Typography>
+                  <Box
+                    sx={{
+                      position: 'absolute',
+                      top: '30%',
+                      left: '10%',
+                      display: 'flex',
+                      margin: '10px 0',
+                    }}
+                  >
+                    <Rating
+                      name="half-rating-read"
+                      defaultValue={Number((movie.vote_average / 2).toFixed(2))}
+                      precision={0.5}
+                      readOnly
+                    />
+                    <Typography ml={1}>{(movie.vote_average / 2).toFixed(2)}</Typography>
+                  </Box>
+
+                  <img
+                    className="poster"
+                    src={`https://image.tmdb.org/t/p/w500/${movie.backdrop_path}`}
```

Comment:
Use `https://image.tmdb.org/t/p/original/${movie.backdrop_path}` instead to get better resolution picture. We should probably try to limit fetching hi res images as it could slow our app